src/werkzeug/wrappers/base_response.py

```diff
@@ -490,16 +491,40 @@ def set_cookie(
             ),
         )
 
-    def delete_cookie(self, key, path="/", domain=None):
+    def delete_cookie(
+        self,
+        key,
+        path="/",
+        domain=None,
+        secure: bool = False,
+        httponly: bool = False,
+        samesite: Optional[str] = None,
```

Comment:
@teymour-aldridge I think `Union[Literal["Strict"], Literal["Lax"], Literal["none"]]` might make sense longer term (not sure if Literal is in 3.6 though).

Reply:
According to the API reference [1](https://docs.python.org/3.8/library/typing.html#typing.Literal) [2](https://www.python.org/dev/peps/pep-0586/) for the typing module it's only been introduced in Python 3.8. I guess we can add a comment making a not of this?

Reply:
I think we can see what the best practice is when we can use it (rather than making a note). I'm hoping `"Strict" | "Lax" | "none"` will be possible.